test/stage2/cbe.zig

```diff
@@ -0,0 +1,85 @@
+const std = @import("std");
+const TestContext = @import("../../src-self-hosted/test.zig").TestContext;
+
+// These tests should work with all platforms, but we're using linux_x64 for
+// now for consistency. Will be expanded eventually.
+const linux_x64 = std.zig.CrossTarget{
+    .cpu_arch = .x86_64,
+    .os_tag = .linux,
+};
+
+pub fn addCases(ctx: *TestContext) !void {
+    ctx.c11("empty start function", linux_x64,
```

Comment:
This doesn't need to be a linux_x64 case right? It can be the default, native target.

Reply:
It goes through the normal Zig -> ZIR pipelines, so I wasn't sure.